Makefile

```diff
@@ -6,7 +6,13 @@ clean:
 	@find . -name "__pycache__" -delete
 
 test:
-	pipenv run python backend/manage.py test $(ARG) --parallel --keepdb
+	python backend/manage.py test $(ARG) --parallel --keepdb
+
+dockertest:
+	docker-compose run backend python backend/manage.py test $(ARG) --parallel --keepdb
```

Comment:
use `--rm` here as this will keep on creating containers on consecutive executions